episodes/03-audience.md

```diff
@@ -89,6 +171,29 @@ to help you assess whether a respondent falls within the intended audience for y
 
 ::::::::::::::::::::::::::::::::::::::::::::::::::
 
+## Defining Prerequisite Knowledge
+
+A very common challenge encountered in workshops is
+heterogeneity of expertise among the audience.
+When learners arrive at a workshop with 
+a wide range of previous experience with the topic,
+it is difficult for the instructors to keep everyone engaged.
+Those who arrive with too little relevant knowledge and experience
+can struggle to follow the lesson content at the pace you expect,
+while those who arrive with too much are likely to become bored 
+and despondent as their expectation of learning new skills is not met.
+
+One way to try to guard against this is to publish 
+the description of your target audience when you advertise a workshop teaching your lesson,
+alongside a list summarising the skills and conceptual knowledge 
+you expect learners to arrive with.
```

Comment:
We could link to some examples of target audience and prerequisite knowledge here (e.g. from core lessons or even Jekyll lesson if we can use lessons from the Incubator) and break the paragraph and introduce another heading named "Testing Prerequisite Knowledge" where we would continue with "Another is to use the information you have about your target audience to ask questions ...", again give some examples of quizes and mention its downsides (Dunning-Kruger effect). For quiz examples we could use e.g. https://carpentries-incubator.github.io/python-intermediate-development/quiz/index.html if it is appropriate to use a lesson from the Incubator.

Reply:
Good suggestion, thanks. I will have a go at adding this, and we should definitely caution against relying too much on self-assessment, but I want to steer clear of mentioning the D-K effect directly, on the grounds that there is [some doubt about whether the effect is actually real](https://www.mcgill.ca/oss/article/critical-thinking/dunning-kruger-effect-probably-not-real)

Reply:
addressed in 6f78866